presidio_evaluator/data_generator/faker_extensions/data_objects.py

```diff
@@ -1,7 +1,8 @@
 from dataclasses import dataclass
```

Comment:
Changes here: 
- added sample_id param to be able to track back sample
- ruff (formatting)